### ISSUE TITLE
feature request: JSON output

### ISSUE THREAD

**Issue by tstromberg** [google/pullsheet#5]
For all current kinds

**Comment by cpanato** [comment]
will work on that

**Comment by bharathk005** [comment]
@tstromberg Is this a requirement to have an option to output a json instead of CSV

**Comment by cpanato** [comment]
did not have much time :( if you want to work on this @bharathk005 go for it

**Comment by bharathk005** [comment]
@cpanato , I would like it. Please let me know if my understanding of the requirement is correct

**Comment by cpanato** [comment]
yes, this might be a new flag that you can choose with the default one or json, if you don't set it the output will be the current implementation

**Comment by bharathk005** [comment]
@cpanato please confirm if we can have the below JSON output for this CSV
CSV output
URL,Date,Project,Reviewer,PRAuthor,PRComments,ReviewComments,Words,Title
https://github.com/kubernetes/minikube/pull/11638,2021-06-11,minikube,spowelljr,sharifelgamal,0,1,12,site: Cleanup FAQ formatting and grammar.
https://github.com/kubernetes/minikube/pull/11618,2021-06-10,minikube,medyagh,felipecrescencio,1,0,10,site: add alternative certs dir

Json output
[
{
	"URL": "https://github.com/kubernetes/minikube/pull/11638",
	"Date":"2021-06-11",
	"Project": "minikube",
	"Reviewer":"spowelljr",
	"PRAuthor": "sharifelgamal",
	"PRComments":"o",
	"ReviewComments": "1",
	"Words":"12",
	"Title": "site: Cleanup FAQ formatting and grammar"
},
{
	"URL": "https://github.com/kubernetes/minikube/pull/11618",
	"Date":"2021-06-10",
	"Project": "minikube",
	"Reviewer":"medyagh",
	"PRAuthor": "felipecrescencio",
	"PRComments":"1",
	"ReviewComments": "0",
	"Words":"10",
	"Title": "site: add alternative certs dir"
}
]

**Comment by cpanato** [comment]
lgtm

**Comment by bharathk005** [comment]
@cpanato @tstromberg 
Created PR #40
